Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 def configurations = [
-    [ platform: "linux", jdk: "8", jenkins: "2.245-rc30116.0ed391051584" ],
-    [ platform: "linux", jdk: "11", jenkins: "2.245-rc30116.0ed391051584", javaLevel: "8" ]
+    [ platform: "linux", jdk: "8", jenkins: "2.245-rc30129.40dba11c77cc" ],
```

Comment:
You shouldn’t need to set Jenkins version here and in your Pom, you can pass null to have it inherited or don’t specify it

Reply:
Thank you! I wasn't aware of that